sled-agent/src/services.rs

```diff
@@ -4325,6 +4310,7 @@ impl ServiceManager {
             return Ok(());
         };
 
+        let log = self.inner.log.new(o!("stage" => SWITCH_ZONE_INIT_STAGE));
```

Comment:
The loggers in this file make a ton of sense to me, and seem like they align pretty precisely 1:1 with the label!

Reply:
🙇‍♀️